systests/src/main.rs

```diff
@@ -0,0 +1,9 @@
+use tests::{meshnet::test_meshnet_poc, vpn::test_vpn_poc};
+
+mod tests;
+mod utils;
+
+fn main() {
```

Comment:
why don't we use the tests approach here? 
```rust
#[cfg(test)]
mod tests_vpn {
    use super::*;

    #[test]
    fn test_vpn_poc_works() {
        test_vpn_poc();
    }
}
```
then we could get nicer test result reports like
```rust
thread 'test_vpn::test_vpn_poc_works' panicked at systests/src/tests/vpn.rs:51:5:
assertion failed: false
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
Successfully executed 'ip link delete tun10'
Successfully executed 'ip rule del priority 32111'


failures:
    test_vpn::test_vpn_poc_works

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 1.22s
```